### ISSUE TITLE
Screenshot capturing doesn't work with hardware bitmaps.

### ISSUE THREAD

**Issue by Laimiux** [Instabug/Instabug-Android#282]
### Steps to Reproduce the Problem
1. Use https://github.com/coil-kt/coil to fetch a hardware bitmap. 
2. After shaking, try to capture the screenshot

### Expected Behavior
The screenshot should be captured and it should go back to report form.

### Actual Behavior
Hides the screenshot button and nothing else happens.

### Instabug integration code
```kotlin
        Instabug.Builder(application, "lalala")
            .setInvocationEvents(InstabugInvocationEvent.SHAKE, InstabugInvocationEvent.SCREENSHOT)
            .setConsoleLogState(Feature.State.ENABLED)
            .build()

        Instabug.setWelcomeMessageState(WelcomeMessage.State.DISABLED)
        Instabug.setDebugEnabled(true)
        BugReporting.setShakingThreshold(100)
```
### SDK Version
`9.0.5`

### Android Version
Android 10

### Device Model
Pixel 3

**Comment by Laimiux** [comment]
From looking into `Instabug` code, it looks like it needs to use `PixelCopy` class instead of using `Canvas` to draw the view.

https://stackoverflow.com/questions/58314397/java-lang-illegalstateexception-software-rendering-doesnt-support-hardware-bit

**Comment by mzelzoghbi** [comment]
Hi Laimonas, 
Thanks for reporting this. We are looking into this and will get back to you **ASAP**.

Thank You!

**Comment by Laimiux** [comment]
Any updates?

**Comment by mzelzoghbi** [comment]
Hi Laimiux,

First of all, please accept our sincerest apologies for taking some time to come up with a fix for your issue. We are still working on a fix to make sure that it is stable and doesn't have any regressions.

We will keep you updated once the fix is ready.
Thank You 😊

**Comment by Laimiux** [comment]
Hi, could you provide an ETA on when this fix will be available?

**Comment by mzelzoghbi** [comment]
Hi @Laimiux ,

We created a **SNAPSHOT** that should fix the issue, Can you give it a try & let us know if it fixed the issue for you?
```
implementation 'com.instabug.library:instabug:9.0.5.2475-SNAPSHOT'
```
you will need to add these lines to your app **build.gradle** before your dependencies
```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
```
This **SNAPSHOT** is completely safe to use and has no debug code at all.

Thank You 😊

**Comment by Laimiux** [comment]
It's working, thanks! Do you know when it will make it into the new release?

**Comment by mzelzoghbi** [comment]
Happy to hear everything worked fine 🎉 It will be included in the release that is coming in ~ 2-3 weeks as we are releasing a new version in a couple of days and it is so tight to include this fix in it.

But once we release the new release I'll send you another snapshot from the latest version. and will let you know once it is included in the final release.

Please let me know if you need any other help.
Thank You 😊

**Comment by Laimiux** [comment]
Hey, we are seeing `Could not resolve com.instabug.library:instabug:9.0.5.2475-SNAPSHOT` in our builds. Were there any changes to snapshot releases?

**Comment by Muhammadyoussef** [comment]
Hi @Laimiux,

I just tested consuming this snapshot and everything works as expected. Please make sure to add the following to your app **`build.gradle`**
```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
```

**Comment by Laimiux** [comment]
@Muhammadyoussef when is this going to make it into the stable release?

**Comment by Laimiux** [comment]
@mzelzoghbi You mentioned that this will make it into a release in 2-3 weeks, however it still doesn't seem to be released. Using the snapshot is making our builds unstable (sometimes dependency doesn't resolve). Could you provide more information when this will be added to which release?

**Comment by mzelzoghbi** [comment]
Hi @Laimiux  ,
Please accept our apologies for the inconvenience we may have caused. It is currently in the testing phase and should be released within next week.

I'll make sure to update you once it is released or if there are any unplanned delays.

Thank You

**Comment by mzelzoghbi** [comment]
Hi @Laimiux 
This issue has been released in 9.1.5. Please give it a try and let us know if you faced any other issues.
Thank You.
